test/mmap1.c

```diff
@@ -0,0 +1,45 @@
+// _DEFAULT_SOURCE for mkstemp  (WHY?)
+#define _DEFAULT_SOURCE
```

Comment:
You also need here:
> `#define _GNU_SOURCE  /* Needed for RTLD_NEXT */`
At least, that's what I'm seeing for WSL Ubuntu 22.04.

Reply:
Fixed.